renaming_status.txt

```diff
@@ -0,0 +1,81 @@
+PROPOSED GITHUB REPO RENAME:    py-ipfs-api -> py-ipfs-http-client
+PROPOSED PIP MODULE RENAME:     ipfshttpclient
+PROPOSED PROJECT RENAME:        Python IPFS API -> Python IPFS HTTP Client
+External Changes to be made:
+(See the section under "FOR README.md")
+
+
+THESE MAY REFER TO THE ACTUAL API
+ipfshttpclient/client.py          All occurrences
+ipfshttpclient/encoding.py        Ln 392
+ipfshttpclient/http.py            Ln 2,4,134
+ipfshttpclient/version.py         Ln 7
+test/run-tests.py                 Ln 68,102
+
+
+FOR README.md:
+* I have made changes in the original file itself
+* External changes to be done by the project maintainers (I have already made the changes in the file, on the respective lines):
+    https://img.shields.io/pypi/v/ipfsapi.svg?style=flat-square -> https://img.shields.io/pypi/v/ipfshttpclient.svg?style=flat-square   line 6
+    https://pypi.python.org/pypi/ipfsapi -> https://pypi.python.org/pypi/ipfshttpclient                                                 line 6
```

Comment:
Will need to be manually just after this branch is merged. Needs transitional code for old package in place.